README.md

```diff
@@ -35,84 +35,56 @@ import "github.com/swaggo/files" // swagger embed files
 ```
 
 ### Canonical example:
-
+Now assume you have implemented a simple api as following:
 ```go
-package main
-
-import (
-	"github.com/gin-gonic/gin"
-	swaggerFiles "github.com/swaggo/files"
-	ginSwagger "github.com/swaggo/gin-swagger"
-
-	_ "github.com/swaggo/gin-swagger/example/basic/docs" // docs is generated by Swag CLI, you have to import it.
-)
+//A get function which returns a Helloworld for you 
+func Helloworld(g *gin.Context)  {
+	g.JSON(http.StatusOK,"helloworld")
+}
 
-// @title Swagger Example API
-// @version 1.0
-// @description This is a sample server Petstore server.
-// @termsOfService http://swagger.io/terms/
+```
+So how to use gin-swagger on api above? Just follow the folling guide:)  
```

Comment:
What do you mean by "folling"?

Reply:
It's `following`, so sorry for typing a typo here:((